server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -2424,6 +2429,17 @@ public NetworkResponse createNetworkResponse(ResponseView view, Network network)
                 response.setVpcName(vpc.getName());
             }
         }
+
+        final NetworkDetailVO detail = networkDetailsDao.findDetail(network.getId(), Network.AssociatedNetworkId);
+        if (detail != null) {
+            Long associatedNetworkId = Long.valueOf(detail.getValue());
+            NetworkVO associatedNetwork = ApiDBUtils.findNetworkById(associatedNetworkId);
+            if (associatedNetwork != null) {
+                response.setAssociatedNetworkId(associatedNetwork.getUuid());
+                response.setAssociatedNetworkName(associatedNetwork.getName());
+            }
+        }
+
```

Comment:
We could invert this `if` statement and add a return, to reduce code indentation.